### PR TITLE
Branding overlay images to not slash faces in half

### PR DESCRIPTION
**Pull request by paperboyo**
Hello,

## What does this change?
Crops sharing images a bit better.
## What is the value of this and can you measure success?
Success is overrated. This change means that some stuff will look a bit less naff. Sometimes.
## Does this affect other platforms - Amp, Apps, etc?
Amp carries og:image and twitter:image, so yes.

---

I don't know what I'm doing and have not tested anything (this extra comma there won't brake anything, won't it?). I'm not even sure where this what I'm writing will appear after I hit Propose file change! [EDIT: now I know and I can't remove it from there 😆]

Currently, social images may crop off important image parts because they are of different ratio than the original crops like so:

[Current](https://i.guim.co.uk/img/media/6ede56d85ef9343b64cb49ca7aa6c36c350c85c8/0_187_2233_2791/master/2233.jpg?w=1200&h=630&q=55&auto=format&usm=12&fit=crop&bm=normal&ba=bottom%2Cleft&blend64=aHR0cHM6Ly91cGxvYWRzLmd1aW0uY28udWsvMjAxNi8wNS8yNS9vdmVybGF5LWxvZ28tMTIwMC05MF9vcHQucG5n&s=016f6a13b8264742dd4512bab44334b1)
2233_current

I propose to introduce additional [imgIX params](https://docs.imgix.com/apis/url/size/crop) only to this sharing image type, so that, hopefully, important image parts won't be cropped off, like so:

[Proposed](https://i.guim.co.uk/img/media/6ede56d85ef9343b64cb49ca7aa6c36c350c85c8/0_187_2233_2791/master/2233.jpg?w=1200&h=630&q=55&auto=format&usm=12&fit=crop&crop=faces,entropy&bm=normal&ba=bottom%2Cleft&blend64=aHR0cHM6Ly91cGxvYWRzLmd1aW0uY28udWsvMjAxNi8wNS8yNS9vdmVybGF5LWxvZ28tMTIwMC05MF9vcHQucG5n&s=016f6a13b8264742dd4512bab44334b1)
2233_proposed

This will inevitably mean some other errors in crops, but I've tested images without faces and it was never worse than the current output (tested also `edges` and inconclusively settled on `entropy`).

@katebee: because it's all her fault for suggesting I should look at code. Oh, and @paperboyo: stay, the f*&%k away with your non-PR-related comments, okay?!

Scared to death
Mateusz

[Heck, let's see what happens when I click this thingy here!]

### CODE REVIEW

**Comment by sndrs** [comment]
YEAH! :goldendiamond:

File: common/app/views/support/Profile.scala [hunk 1]
```diff
@@ -104,13 +104,14 @@ object Video700 extends VideoProfile(width = Some(700), height = Some(394)) // 1
 abstract class ShareImage(shouldIncludeOverlay: Boolean) extends Profile(width = Some(1200)) {
   override val heightParam = "h=630"
   override val fitParam = "fit=crop"
+  val cropParam = "crop=faces,entropy"
```
Comment (sndrs) [inline common/app/views/support/Profile.scala]: @rich-nguyen see the `entropy` literally creeping in

**Review by sndrs — APPROVED** [review]

**Comment by sndrs** [comment]
@phamann it's happened...

**Comment by phamann** [comment]
@paperboyo Probably my favourite PR description in history. Ever.

> Success is overrated. This change means that some stuff will look a bit less naff. Sometimes.

> I don't know what I'm doing and have not tested anything

> Scared to death

> Heck, let's see what happens when I click this thingy here!

:joy:

PLUS 1

**Comment by gtrufitt** [comment]
ping the @jfsoul man

**Comment by katebee** [comment]
👍 Fantastic! 🎉 

Want me to merge and deploy?

File: common/app/views/support/Profile.scala (hunk 1 above)
Comment (rich-nguyen) [inline common/app/views/support/Profile.scala]: ⏳

**Comment by phamann** [comment]
> Come back - amateurs are taking over the site!!11ONE1!!

@paperboyo At least it's you Matt and not @gklopper

**Comment by gklopper** [comment]
@phamann @paperboyo nobody gives me a 👍 anymore, my work here is done...

**Comment by rich-nguyen** [comment]
@gklopper there is nothing here to 👍 
screen shot 2016-12-02 at 15 54 23

**Comment by phamann** [comment]
Just gonna leave this here:

google chrome 21

**Pull request merged**